docs/concepts/endpoint.md

```diff
@@ -0,0 +1,78 @@
+---
+id: endpoint
+title: Endpoint
+---
+
+Endpoints in ZIO HTTP are defined using the `Endpoint` object's combinators, which provide a type-safe way to specify various aspects of the endpoint. For instance, consider defining endpoints for retrieving user information and user posts:
+
+```scala
```

Comment:
Please make sure all of your code snippets are compile-safe [by adding `mdoc` modifiers](https://scalameta.org/mdoc/docs/modifiers.html).

Reply:
noted